src/apis/servicing_api.ts

```diff
@@ -157,6 +157,20 @@ export class ServicingAPI {
         );
     }
 
+    public async getTotalExpectedRepayment(issuanceHash: string): Promise<BigNumber> {
+        this.assert.schema.bytes32("issuanceHash", issuanceHash);
+
```

Comment:
Please add assertion that the debt agreement exists + a test for the failed scenario -- i.e. using `this.assert.debtAgreement.exists`.